nengo_loihi/emulator/interface.py

```diff
@@ -5,7 +5,7 @@
 import warnings
 
 from nengo.exceptions import SimulationError
-from nengo.utils.compat import iteritems, itervalues, range
```

Comment:
Why remove `range`?

Reply:
In Python 2, we do this import so that `range` is `xrange`, which gives an iterator instead of generating the full list, which can be expensive. In Python 3, `range` is already an iterator, and there is no `xrange` in Python 3. Since Nengo Loihi is Python 3 only, we can just use `range`.

Reply:
As for practically why I did it, I was adding some imports there while working on this, and with those new imports it broke onto a new line, so I was like ... might as well just remove range and it'll stay on one line.

Reply:
Can't we get rid of `iteritems` and `itervalues` then, too? Also, if we're making these changes, I'd do them throughout the repo.

Reply:
Yes, I forget where but at one point I grepped and get rid of all of the `iteritems` but either missed some or some were reintroduced with rebasing and whatnot. I'll do that now.

Reply:
Yeah the only ones remaining were in `emulator/interface.py` which was introduced after I did that; done in58c448fa63ce5c71692715f451e351c8514a8fd3.